cinder/volume/drivers/nexenta/nfs.py

```diff
@@ -107,12 +104,13 @@ def __init__(self, *args, **kwargs):
                               'option': option})
                 raise jsonrpc.NmsException(code='EINVAL', message=message)
         self.nms = None
+        self.nas_root = None
```

Comment:
```suggestion
        self.nas_root_stats = None
```

Reply:
or maybe even self.nas_path_stats

Reply:
Changed to self.nas_stat and self.san_stat, thank you!